main.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable no-unused-vars */
```

Comment:
Move this to the project config if it is needed for the entire project. Or move it to the statement that is violating this rule. Or fix the statement that is violating the rule.